### PR TITLE
docs: Update roadmap for the v1.11 release

### PR DESCRIPTION
**Pull request by travisn**
**Description of your changes:**
The roadmap is updated with planned features for the v1.11 release targeted for January.

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Review by BlaineEXE — CHANGES_REQUESTED** [review]

**Review by subhamkrai — COMMENTED** [review]
Should we include the krew issue too which we are planning to have by rook 1.11? It will not be a hard requirement for release but good to have it on track.

**Comment by subhamkrai** [comment]
> Should we include the krew issue too which we are planning to have by rook 1.11? It will not be a hard requirement for release but good to have it on track.

if we decide to include, https://github.com/rook/kubectl-rook-ceph/issues/68 and https://github.com/rook/kubectl-rook-ceph/issues/69 are good to track IMO

File: ROADMAP.md [hunk 1]
```diff
@@ -8,28 +8,26 @@ We hope that the items listed below will inspire further engagement from the com
...
+The following high level features are targeted for Rook v1.11 (January 2023). For more detailed project tracking see the [v1.11 board](https://github.com/rook/rook/projects/27).
 
-* Remove support for Ceph Octopus (support remains for Pacific and Quincy) [#10338](https://github.com/rook/rook/issues/10338)
-* Add command to the [krew plugin](https://github.com/rook/kubectl-rook-ceph) to analyze cluster health and advise on resolving common health issues [Krew #32](https://github.com/rook/kubectl-rook-ceph/issues/32)
-* Check for existing subvolumes before allowing a filesystem to be uninstalled [#9915](https://github.com/rook/rook/pull/9915)
 * Use more specific cephx accounts to better differentiate the source of Ceph configuration changes [#10169](https://github.com/rook/rook/issues/10169)
+* Enable restoring a cluster from the OSDs after all mons are lost [#7049](https://github.com/rook/rook/issues/7049)
 * Automate node fencing for application failover in some scenarios [#1507](https://github.com/rook/rook/issues/1507)
+* Support RBD mirroring across clusters with overlapping networks [#11070](https://github.com/rook/rook/issues/11070)
```
Comment (Madhu-1) [inline ROADMAP.md:18]: Is this also applicable for CephFS mirroring?

Comment (travisn) [reply ROADMAP.md:18]: Yes, the same would apply to cephfs mirroring, so it's just a scenario priority to say we first want to make sure rbd mirroring is handled.

File: ROADMAP.md [hunk 2]
```diff
@@ -8,28 +8,26 @@ We hope that the items listed below will inspire further engagement from the com
...
+* OSD encryption on partitions [#10984](https://github.com/rook/rook/issues/10984)
 * Object Store
   * Service account authentication with the Vault agent [#9872](https://github.com/rook/rook/pull/9872)
-  * Support for AWS Server Side Encryption (SSE) [#10318](https://github.com/rook/rook/pull/10318)
-  * Improvements to RGW Multisite configuration [#10323](https://github.com/rook/rook/pull/10323)
-
+  * Add alpha support for COSI (Container object storage interface) with K8s 1.25 [#7843](https://github.com/rook/rook/issues/7843)
```
Comment (humblec) [inline ROADMAP.md:22]: may be 1.26 ?

File: ROADMAP.md [hunk 3]
```diff
@@ -8,28 +8,26 @@ We hope that the items listed below will inspire further engagement from the com
...
+  * Support the immutable object cache [#11162](https://github.com/rook/rook/issues/11162)
```
Comment (Madhu-1) [inline ROADMAP.md:23]: We will have cephfs encryption support https://github.com/ceph/ceph-csi/pull/3460 in cephcsi 3.8.0 , still have some dependencies with ceph

**Review by parth-gr — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Review by galexrt — CHANGES_REQUESTED** [review]
Let's include the main krew plugin tasks and make sure that the COSI support is targeted for the "correct" K8S version

Besides that then LGTM

**Review by galexrt — APPROVED** [review]

**Review by subhamkrai — APPROVED** [review]

**Review by BlaineEXE — APPROVED** [review]

**Pull request merged**